data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -222,7 +222,7 @@
                                     [message]
                                         speaker=narrator
                                         image="wesnoth-icon.png"
-                                        message= _ "You struggle to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. You have increased resistance to all physical damage!"
+                                        message= _ "$unit.name| struggles to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. $unit.name| feels an increased resistance to all physical damage!"
```

Comment:
Any particular reason why the unit wasn't made the speaker here, like it is with the flaming sword? I presume the rationale with the flaming sword was to make sure the unit gets highlighted, and that would apply here as well.

Also something to be aware of: changing the text to refer to "$unit.name" instead of "you" is fine, but it only works if the player can never get a unit which doesn't have a name. I don't think that possibility exists currently in the campaign, though, so this should be safe. And maybe you already considered that, I'm just saying in case it's relevant in the future with similar changes in some other campaign.